src/v3/src/transformer/layout/securityQuestion/securityQuestionEnroll.ts

```diff
@@ -74,7 +74,10 @@ export const transformSecurityQuestionEnroll: IdxStepTransformer = ({ transactio
   const predefinedQuestions = contextualData?.questions?.map((question: Question) => ({
     value: question.questionKey,
     label: question.question,
-  } as IdxOption));
+  } as IdxOption)).map((opt: IdxOption) => ({
+    ...opt,
+    label: loc(`security.${opt.value}`, 'login'),
+  }));
```

Comment:
Fixes english leaks for security question view.

Reply:
A bit wary of the string interpolation in the `loc` function. Is there any other way to do this?

Reply:
The translation keys are all postfixed with the question key, so like `security.disliked_food`, `security.favorite_plush_toy`, etc. 

We will have to iterate through the options to get those which are present in the response to determine which keys are used. Granted if a new key is added (without it being added to translations) it would cause a L10 error, but hopefully that would be caught with leaks detection in okta-core.